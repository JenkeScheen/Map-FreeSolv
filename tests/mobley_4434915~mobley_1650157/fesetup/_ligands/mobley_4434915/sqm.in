Run semi-empirical minimization
 &qmmm
  qm_theory='AM1',grms_tol=0.0002,tight_p_conv=1,
  scfconv=1.0d-10,itrmax=500,pseudo_diag=1,
  maxcyc=1000,
  qmcharge=0,
 /
   6    C1       -0.1150       -0.0080       -0.0770 
  17   Cl2        1.3490        0.0980        0.9060 
   1    H3       -0.9600        0.3810        0.4960 
   1    H4        0.0210        0.5830       -0.9860 
   1    H5       -0.2950       -1.0540       -0.3380 

