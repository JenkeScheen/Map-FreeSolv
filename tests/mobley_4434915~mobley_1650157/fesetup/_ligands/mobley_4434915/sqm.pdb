ATOM      1  C1  LIG     1      -0.115  -0.008  -0.077  1.00  0.00           C
ATOM      2 Cl2  LIG     1       1.327   0.097   0.892  1.00  0.00          Cl
ATOM      3  H3  LIG     1      -0.965   0.391   0.518  1.00  0.00           H
ATOM      4  H4  LIG     1       0.038   0.597  -0.997  1.00  0.00           H
ATOM      5  H5  LIG     1      -0.285  -1.076  -0.335  1.00  0.00           H
