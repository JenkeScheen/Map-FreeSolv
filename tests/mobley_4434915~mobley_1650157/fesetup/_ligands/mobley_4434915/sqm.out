            --------------------------------------------------------
                             AMBER SQM VERSION 14
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =  7

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: Cl: M.J.S.DEWAR et al. THEOCHEM, 180, 1, (1988)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.1D-09
| QMMM: Maximum number of SCF cycles :      500

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       11
| QMMM:    diag iterations used for timing =       20
| QMMM:
| QMMM:              Internal diag routine = 0.000152 seconds
| QMMM:                 Dspev diag routine = 0.000887 seconds
| QMMM:                Dspevd diag routine = 0.000301 seconds
| QMMM:                Dspevx diag routine = 0.000761 seconds
| QMMM:                 Dsyev diag routine = 0.000300 seconds
| QMMM:                Dsyevd diag routine = 0.000305 seconds
| QMMM:                Dsyevr diag routine = 0.000291 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.000030 seconds
| QMMM:
| QMMM: Using internal diagonalization routine (diag_routine=1).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C       -0.1150   -0.0080   -0.0770
  QMMM:     2        2      Cl       1.3490    0.0980    0.9060
  QMMM:     3        3      H       -0.9600    0.3810    0.4960
  QMMM:     4        4      H        0.0210    0.5830   -0.9860
  QMMM:     5        5      H       -0.2950   -1.0540   -0.3380

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -37.2685  -28.8467  -15.1495  -15.1495  -14.8668  -11.3381  -11.3381    1.5978
   3.8650    3.8650    4.1959

 Heat of formation   =        -18.97692043 kcal/mol  (       -0.82290102 eV)

 Total SCF energy    =     -12533.00297230 kcal/mol  (     -543.47179100 eV)
 Electronic energy   =     -23151.17513050 kcal/mol  (    -1003.91028709 eV)
 Core-core repulsion =      10618.17215820 kcal/mol  (      460.43849608 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C                 -0.177
     2      Cl                -0.117
     3      H                  0.098
     4      H                  0.098
     5      H                  0.098
 Total Mulliken Charge =      -0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    -1.254   -0.091   -0.842    1.513
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C       -0.1153   -0.0084   -0.0772
  QMMM:     2        2      Cl       1.3275    0.0966    0.8918
  QMMM:     3        3      H       -0.9648    0.3910    0.5181
  QMMM:     4        4      H        0.0376    0.5970   -0.9968
  QMMM:     5        5      H       -0.2852   -1.0762   -0.3349

           --------- Calculation Completed ----------

